flutter/test/event_processor/flutter_enricher_event_processor_test.dart

```diff
@@ -22,7 +22,38 @@ void main() {
       fixture = Fixture();
     });
 
-    testWidgets('flutter context', (WidgetTester tester) async {
+    testWidgets('flutter context on dart:io', (WidgetTester tester) async {
+      if (kIsWeb) {
+        // widget tests don't support onPlatform config
```

Comment:
Do you have any better idea than this workaround?

Reply:
Hm, don't think so but I'm fine with this as long as it's documented